problems/lib/problem2.rb

```diff
@@ -0,0 +1,26 @@
+class Problem2
+  def get_largest_two_numbers(array)

```

Comment:
Yeah... it works...
But it's one big chunk of code that if converted in C wouldn't look much different :(
Ruby should be pleasant to the eye, actually all code should be...

As Kent Beck says : "Make it work, make it right, make it fast"
How would you refactor this code to be more readable, with less duplication in concepts and with less assignments